src/axolotl/integrations/diffusion/README.md

```diff
@@ -0,0 +1,125 @@
+# Diffusion LM Training Plugin for Axolotl
+
+This plugin enables diffusion language model training using the LLaDA (Large Language
+And Diffusion Assistant) approach within the Axolotl framework.
+
+## Overview
+
+LLaDA is a diffusion-based approach to language model training that uses:
+- **Random token masking** during training instead of next-token prediction
+- **Bidirectional attention** to allow the model to see the full context
+- **Importance weighting** based on masking probabilities for stable training
+
+This approach can lead to more robust language models with better understanding of
+bidirectional context.
+
+## Installation
+
+The plugin is included with Axolotl. To use it, simply add the plugin configuration to
+your training config.
+
+## Quickstart
```

Comment:
For this Quickstart, it may be easier to start with 
1. Install axolotl via (link to doc)
2. Run `axolotl train examples/llama-3/diffusion...` for pretrain/sft (maybe similar to the gpt-oss readme)

This way, users can be up and running within 2-3 steps.

Then, the section below can explain the configurations like you've done.

It can also help to have a link to the API schema for the pydantic schema for this plugin if possible. I think it'll be
```
https://docs.axolotl.ai/docs/api/integrations.diffusion.args.html#axolotl.integrations.diffusion.args
```

Edit: Seems like the arg classes are not rendered also for other plugins.